ci/deployment.yaml

```diff
@@ -55,15 +55,18 @@ spec:
           ports:
             - containerPort: 5000
           volumeMounts:
-            - mountPath: /secrets/oauth-token
-              readOnly: true
-              name: hail-ci-0-1-github-oauth-token
-            - mountPath: /ci-jwt
-              readOnly: true
-              name: ci-jwt
-            - name: jwt-secret-key
-              mountPath: /jwt-secret-key
-              readOnly: true
+           - name: ci-user-secret
+             mountPath: /ci-user-secret
+             readOnly: true
```

Comment:
if you make changes anyway: symmetry: readOnly first, and maybe fix `jwt-secret-key` to be the same way